src/firecracker/src/main.rs

```diff
@@ -159,6 +160,20 @@ fn main() {
             if let Some(version) = arg_parser.arguments().value_as_bool("version") {
                 if version {
                     println!("Firecracker v{}\n", FIRECRACKER_VERSION);
+
+                    print!("Supported snapshot data format versions:");
```

Comment:
nit: I think it would be clearer to have a separate function that prints the snapshot versions

Reply:
Done